### PR TITLE
feat: hex string to number

### PR DESCRIPTION
**Pull request by shaojunda**

### CODE REVIEW

File: app/models/block.rb [hunk 1]
```diff
@@ -121,8 +118,11 @@ def invalid!
 #  dao                        :string
 #  primary_reward             :decimal(30, )    default(0)
 #  secondary_reward           :decimal(30, )    default(0)
-#  nonce                      :string
 #  chain_root                 :binary
+#  difficulty                 :decimal(30, )    default(0)
```
Comment (classicalliu) [inline app/models/block.rb]: `difficulty` is an U256, larger than `decimal(30)`

Comment (shaojunda) [reply app/models/block.rb]: thanks, I will fix this.

**Comment by shaojunda** [comment]
test failed because SDK return `chain_root`, after PR: https://github.com/nervosnetwork/ckb-sdk-ruby/pull/179 merged, I will fix test

**Comment by shaojunda** [comment]
please ignore git actions.

**Pull request merged**